src/api/movies.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+export const API_BASE_URL = "http://localhost:3000";
+
+export const getMovies = async () => {
+  return await axios.get(`${API_BASE_URL}/movies`);
```

Comment:
Since `/movies` is a resource it could be extracted from here and set as `const` so you'll be able to use it also in other requests when you'll add them.